.github/workflows/python-app.yml

```diff
@@ -23,15 +23,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
we don't use flake, so just removed that.. added a separate action for black formatting